differ/storage_test.go

```
@@ -2483,3 +2483,233 @@ func TestWriteNotificationRecordForClusterWrongDriver(t *testing.T) {
+		RuleID:    "test_rule",
+		ErrorKey:  "TEST_RULE_IMPORTANT_IMPACT",
+	}
+	_, exists = disabledRules[key2]
+	assert.True(t, exists, "Important impact rule should be present")
+
+	// connection to mocked DB needs to be closed properly
+	checkConnectionClose(t, connection)
+
+	// check if all expectations were met
+	checkAllExpectations(t, mock)
+}
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)